optuna/samplers/_tpe/sampler.py

```diff
@@ -132,6 +142,8 @@ def __init__(
         gamma: Callable[[int], int] = default_gamma,
         weights: Callable[[int], np.ndarray] = default_weights,
         seed: Optional[int] = None,
+        *,
+        multivariate: bool = False
```

Comment:
Seems like a good interface for exposing this feature to the user, not introducing any new concepts besides an argument 👍